src/Serilog/Rendering/MessageTemplateRenderer.cs

```diff
@@ -68,7 +68,7 @@ public static void RenderPropertyToken(PropertyToken pt, IReadOnlyDictionary<str
             return;
         }
 
-        var valueOutput = new StringWriter();
+        using var valueOutput = ReusableStringWriter.GetOrCreate();
         RenderValue(propertyValue, isLiteral, isJson, valueOutput, pt.Format, formatProvider);
         var value = valueOutput.ToString();
```

Comment:
Note for subsequent PR - avoid such `ToString()` calls by using `StringBuilder` from `valueOutput` + add `Padding.Apply` overload that accepts `StringBuilder`.